etna/auto/auto.py

```diff
@@ -223,7 +206,7 @@ def __init__(
         backtest_params:
             custom parameters for backtest instead of default backtest parameters
         experiment_folder:
-            folder to store experiment results and name for optuna study
+            name for saving experiment results, it determines the name for optuna study
```

Comment:
We should probably clarify here about the default value. And for other parameters too, because it isn't really obvious.